dev-manuals/test-plans/overworld-frontend/plans/npc-load-dialogue.md

```diff
@@ -0,0 +1,37 @@
+# Name: NPC load dialogue
+
+Number: U.01.2
```

Comment:
Didn't we say to make the number the test case identifier?
So that i.e. this testplans number would be `U.NPC.2`?